src/rgw/rgw_op.h

```diff
@@ -1065,7 +1065,7 @@ class RGWGetACLs : public RGWOp {
 
 class RGWPutACLs : public RGWOp {
 protected:
-  size_t len;
+  int len;
```

Comment:
we reuse len in {xml, json} parser.parse, which again expects len as an int & although `recv_body` originally returns a size_t, we return an int in `RGWRestfulIO::recv_body` which uses the int to reraise the exception